pkg/client/portforward/portforward_test.go

```diff
@@ -206,6 +206,102 @@ func (s *fakeUpgradeStream) Headers() http.Header {
 	return http.Header{}
 }
 
+func TestGetListener(t *testing.T) {
+	stopChan := make(chan struct{}, 1)
+	pf, err := New(&client.Request{}, &client.Config{}, []string{"15000", "15000"}, stopChan)
+	if err != nil {
+		t.Logf("Cannot create PortForwarder %v: %v", pf, err)
```

Comment:
This should be `t.Fatalf` as the remainder of the test code won't run without a PortForwarder